exercises/practice/gigasecond/lib/gigasecond.ex

```diff
@@ -3,8 +3,7 @@ defmodule Gigasecond do
   Calculate a date one billion seconds after an input date.
   """
   @spec from({{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}) ::
-          :calendar.datetime()
-
+          {{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}
```

Comment:
The input and output are of the same type, so they should be the same in the type spec. I think `{{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}` is way uglier to write, but more clear to beginners. I'm not sure...

Reply:
🤷🏻 either or, I'm not sure that detail is limiting.